OKTA-447961 Merge the OIN building/submission guides for OIDC integrations (#4146)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

* **SWA apps:** Okta no longer publishes new Secure Web Authentication (SWA) integrations to the OIN catalog. The OIN team maintains existing SWA integrations.

> * **Custom authorization server:** An OIDC or API service integration can't use a [custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server), including the default server. You can only use the [org authorization server](/docs/concepts/auth-servers/#org-authorization-server).

Reviewer: ```suggestion
* **Custom authorization server:** An OIDC or API service integration can't use a [custom authorization server](/docs/concepts/auth-servers/#custom-authorization-server), including the `default` server. You can only use the [org authorization server](/docs/concepts/auth-servers/#org-authorization-server).
```

> * **Okta SDKs and validating access tokens:** The Okta SDKs can't be used to validate access tokens with the [org authorization server](/docs/concepts/auth-servers/#org-authorization-server).

Reviewer: ```suggestion
* **Okta SDKs and validating access tokens:** You can't use the Okta SDKs to validate access tokens with the [org authorization server](/docs/concepts/auth-servers/#org-authorization-server).
```
